.github/workflows/setup-provider-image.yaml

```
@@ -0,0 +1,130 @@
+            TAG="${{ github.event.inputs.tag_name }}"
+          else
+            TAG="commit-$(git rev-parse --short HEAD)"
+          fi
+          echo "TAG_NAME=$TAG" >> "$GITHUB_OUTPUT"
+          echo "TAG=$TAG"       >> "$GITHUB_ENV"
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Prepare platform env
+        run: |
```

actionlint (1.7.7)

got unexpected character '/' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)